letsencrypt/models/acme_tiny.py

```diff
@@ -0,0 +1,199 @@
+# flake8: noqa
```

Comment:
Then the best would be that upstream merges https://github.com/diafygi/acme-tiny/pull/59 and we install this by pip